pkg/controller/container-runtime-config/helpers.go

```diff
@@ -58,17 +61,21 @@ type tomlConfigCRIO struct {
 
 type updateConfigFunc func(data []byte, internal *mcfgv1.ContainerRuntimeConfiguration) ([]byte, error)
 
-func createNewCtrRuntimeConfigIgnition(storageTOMLConfig, crioTOMLConfig []byte) igntypes.Config {
+// createNewIgnition returns an ignition config containing the new data configure by the CRs
+func createNewIgnition(configs map[string][]byte) igntypes.Config {
 	tempIgnConfig := ctrlcommon.NewIgnConfig()
 	mode := 0644
-	// Create storage.conf ignition
-	if storageTOMLConfig != nil {
-		storagedu := dataurl.New(storageTOMLConfig, "text/plain")
+	// Create ignitions
+	for filePath, data := range configs {
+		if data == nil {
+			continue
+		}
+		storagedu := dataurl.New(data, "text/plain")
```

Comment:
This function is a nice generalization — make the name of this variable generic as well, please.

Reply:
Fixed